lib/statusapi.go

```diff
@@ -0,0 +1,62 @@
+package main
+
+// This is a file contains the interface that the package requires
```

Comment:
Following https://golang.org/doc/effective_go.html#commentary this comment should be above `package main`. You can also call this file `types.go` or `interface.go` to better reflect what is in here.